_life/transportation.md

```diff
@@ -58,3 +58,25 @@ If you want to come to EPFL by car, parking rules can be found [here](https://pa
 You have access to [the car sharing service Mobility](http://www.mobility.ch/) through EPFL, with discounts. See [here](https://www.epfl.ch/campus/mobility/vehicles/mobility-carsharing/).
 
 You can also get special discounts with Europcar with a Camipro; see [here](https://support.epfl.ch/epfl?id=epfl_kb_article_view&sysparm_article=KB0014917&sys_kb_id=921965341b1be0d0df3341588b4bcb5a).
+
+### Exchanging a foreign driver’s license
+
+Detailed information about exchanging a foreign driver’s license for a Swiss one is available [on the Canton of Vaud website](https://www.vd.ch/prestation/echanger-un-permis-de-conduire-etranger). Key points:
+* You have 12 months from the date of your arrival in Switzerland to exchange your license.
+* [Depending on the country that issued your license](https://www.vd.ch/themes/mobilite/automobile/permis-de-conduire/echange-permis-etrangers/pays-ayant-une-convention-de-reconnaissance-mutuelle-avec-la-suisse), you may have no additional testing requirements (e.g. for EU countries), only the theoretical test, or both the theoretical and the practical driving test.
+  + USA is on the “only theoretical” list—but this is only because a standard US license (or to be precise, standard licenses from most US states) lets you drive some kinds of light trucks that are not covered by a standard Swiss license.
+    If you are willing to give up that extra privilege, you do not have any testing requirements.
+    This information is not available on the official website but was provided verbally by a staff member so you just have to fill out the form and trust the process.
+    I can confirm through personal experience that this worked for a WA driver’s license in 2023.
+* A section of the form is to be filled out by an optician after an eye exam.
+  You can have it done at many eyeglasses shops.
+  It has a standard price (15 CHF in 2023) and takes just a few minutes.
+
+You may be used to ridiculously long wait times for in-person visits to the motor vehicles office in your home country.
+Switzerland, of course, is not like that.
+When I visited the Lausanne office on a weekday morning, I had to wait for barely ten minutes, and was out the door in another ten.
+
+Exchanging your foreign license typically means that you surrender your old license and get a new Swiss one.
+However, the authorities will only take your old license if they have an agreement with the issuing country or some other means to actually cancel it.
+Absent that, there is no point, so you get to keep it.
+This is true at least for US licenses.
```

Comment:
Maybe add something like "If you exchanged a license from another country and have some useful info, please let us know" ?